pkgs/development/python-modules/dbt-bigquery/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pythonOlder
+, autopep8
+, protobuf
+, google-cloud-core
+, google-cloud-bigquery
+, google-api-core
+, googleapis-common-protos
+}:
+
+buildPythonPackage rec {
+  pname = "dbt-bigquery";
+  version = "0.21.0";
+
+  disabled = pythonOlder "3.7";
+
+  propagatedBuildInputs = [
+    autopep8
```

Comment:
Is this actually a _runtime_ requirement for the package? Not just needed for e.g. the tests?

Reply:
Yes in the setup.py source code (`python_requires=">=3.7"`)

Reply:
No, I'm referring to `autopep8`

Reply:
TBH, I'm not sure, it seems that only the test phase uses it, but I can't find anything about autopep8 or pycodestyle in DBT and all dependencies.